newIDE/app/src/ObjectEditor/Editors/SpriteEditor/SpacedDismissableTutorialMessage.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+import * as React from 'react';
+import { Column, Spacer } from '../../../UI/Grid';
+import useDismissableTutorialMessage from '../../../Hints/useDismissableTutorialMessage';
+
+/**
+ * TODO : Use context directly in SpriteEditor
```

Comment:
Sad but ok there.

Ubernitpicking, no space before colon:
```suggestion
 * TODO: Use context directly in SpriteEditor
```